.github/workflows/CI.yml

```diff
@@ -0,0 +1,113 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - "master"
+      - "maintenance/.+"
+  pull_request:
+    branches:
+      - "master"
+      - "maintenance/.+"
+  schedule:
+    # Nightly tests run on master by default:
+    #   Scheduled workflows run on the latest commit on the default or base branch.
+    #   (from https://help.github.com/en/actions/reference/events-that-trigger-workflows#scheduled-events-schedule)
+    - cron: "0 0 * * *"
+
+jobs:
+  test:
+
+    runs-on: ${{ matrix.cfg.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version:
+          - 3.6
+          - 3.7
+          # Uncomment when ready ;)
+          # - 3.8
+        cfg:
+          - os: ubuntu-latest
+            rdkit: true
+            openeye: false
+
+          - os: ubuntu-latest
+            rdkit: false
+            openeye: true
+
+          - os: ubuntu-latest
+            rdkit: true
+            openeye: true
+
+          - os: macOS-latest
+            rdkit: true
+            openeye: false
+
+    env:
+      CI_OS: ${{ matrix.cfg.os }}
+      RDKIT: ${{ matrix.cfg.rdkit }}
+      OPENEYE: ${{ matrix.cfg.openeye }}
+      PYVER: ${{ matrix.python-version }}
+      OE_LICENSE: ${{ github.workspace }}/oe_license.txt
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Additional info about the build
+      shell: bash
+      run: |
+        uname -a
+        df -h
+        ulimit -a
+    - name: Fix conda permissions in MacOS
+      shell: bash
+      if: startsWith(matrix.cfg.os, 'macOS')
+      run: |
+        sudo chown -R $(id -u):$(id -g) ${CONDA}
+    - name: Configure conda
+      shell: bash
+      run: |
+        . devtools/gh-actions/initialize_conda.sh
```

Comment:
What is the purpose of this `.`? Is it telling GHA to run it out of a particular directory? It obviously works fine, I am just curious.

Reply:
`.` is an alias to `source`. We are importing that initializing script in each step because each GHA step is a separate bash process.